MonoGame.Framework/SDL/SDLGamePlatform.cs

```diff
@@ -27,12 +27,14 @@ public override GameRunBehavior DefaultRunBehavior
 
         private int _isExiting;
         private SdlGameWindow _view;
+        private TextInputEventArgs _textInputEventArgs;
```

Comment:
This makes me wonder... does `TextInputEventArgs` need to be a class?  Can it not be a struct?

Reply:
Reuse of an EventArgs is problematic. After you fire an Event the eventArgs belongs to the user, he would not expect previous EventArgs to change on the fly.
This requires a change in design, using a delegate or registering an ITextInputObserver interface. 
The best we can for now is to skip the creation of TextInputEventArgs  if the user didn't subscribe to the event.

Reply:
Yea, we get it, thats why Tom was suggesting making it a struct...

Reply:
@nkast - i always want to see you involved.  You make good suggestions and have good ideas. :)

Reply:
I try to use my casual speech when replying to stuff that is higher knowledge programming related, I mean no offense.

Reply:
Now that I thought about the code, you should never try to keep EventArgs in the first place, since if anyone/anything else is subscribed to that event, they also get the exact same copy and can modify it.

Reply:
I agree no one should.  But still feel the right fix is to convert `TextInputEventArgs` to a struct.

Reply:
I don't like it, but I've done it.